apps/frontend-pwa/src/components/practiceQuiz/ElementStack.tsx

```
@@ -358,75 +358,77 @@ function ElementStack({
+            className={{ root: '' }}
+            disabled={Object.values(studentResponse).some(
+              (response) => !response.valid
+            )}
+            onClick={() => {
+              // update the read status of all content elements in studentResponse to true
+              setStudentResponse((currentResponses) =>
+                Object.entries(currentResponses).reduce<StackStudentResponseType>(
+                  (acc, [instanceId, value]) => {
+                    if (value.type === ElementType.Content) {
+                      return {
+                        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)